web/webkit/src/main/scala/net/liftweb/http/js/LiftJavaScript.scala

```diff
@@ -75,20 +75,14 @@ object LiftJavaScript {
   }
 
   def initCmd(settings: JsObj): JsCmd = {
-    val extendCmd = LiftRules.jsArtifacts match {
-      case jsa: JQueryArtifacts => Call("window.lift.extend", JsVar("lift_settings"), JsVar("window", "liftJQuery"))
+    val extendJqOrVanilla = LiftRules.jsArtifacts match {
```

Comment:
Yeah, I like yours better.  The original name was so generic as to not be correct because it isn't the only extend.  I over corrected. :)